lightwood/analysis/nc/nc.py

```diff
@@ -251,7 +251,7 @@ def score(self, x, y=None):
         return norm
 
 
-class BaseModelNc(BaseScorer):
+class BaseMixerNc(BaseScorer):
```

Comment:
in theory, can analyzers work on encoder models? this is just to see if we can keep the basemodel nomenclature here, because any *model* can be analyzed, but mixers specifically go after the target.

either way this isnt an issue, just a comment.

Reply:
This particular approach (conformal prediction) should work for any object that generates a prediction given some input. If, say, you used a text encoder as the whole model, then this should work provided an appropriate interface is in place.